### ISSUE TITLE
[Feature] Support for .netcore 2.0

### ISSUE THREAD

**Issue by SergiusZurawski** [GHPReporter/Ghpr.Core#54]
Is there a chance this project will be ported to Dotnet Core 2.0 ?

**Comment by elv1s42** [comment]
Hi @SergiusZurawski !

Thank you for this issue. I agree that it would be useful.
Putting 'enhancement' label to resolve it in the future.

**Comment by Rogitel** [comment]
Any update on this? I love this tool, want to continue using it...

**Comment by Rogitel** [comment]
That would be great! I've been using NUnit.

**Comment by jmemmons** [comment]
Hey @elv1s42, can I go ahead and do this or have you already started?

**Comment by elv1s42** [comment]
Hi @jmemmons,
Sorry for the late response, for some reason I stoped receiving notification emails about the issue.

I didn't start implementation becuase I don't have much time to work on it and this issue is actually quite a big piece of work. There are some open questions around it like "Should we create a separate solution?", "How to run tests for .NET Core project?", "How do we reuse code?", "How to release mutilple versions?" etc

I think we should try re-use existing classes and add try to them as links to .NET Core project.
Also we will have to review AppVeyor file to build, run tests and publish mutliple solutions.

**Comment by jmemmons** [comment]
I went ahead and started porting the code to .NET Core 2.1 in my forked repository to get an idea of what needs to happen.

I created another project called "Ghpr.Core.NetCore" in the same solution and added a new build configuration for .NET 4.7.2 called "Release-NET47". The projects can live together in the same solution which should make development easier.

The only class that gave me issues was `ScreenshotHelper` which uses a library that's not available in .NET Core (System.Drawing). I also had trouble with the embedded resources but I wasn't able to spend much time with it. I'd suggest the first thing we do is to port the tests over first so we have a base regression suite we can use to verify that code is not broken during this process.
